package.json

```diff
@@ -115,10 +115,8 @@
     "ember-source": "~3.14.1",
     "ember-source-channel-url": "^2.0.1",
     "ember-try": "^1.2.1",
-    "eslint": "^4.18.2",
-    "eslint-config-ember": "^0.3.0",
-    "eslint-plugin-ember": "^7.1.0",
-    "eslint-plugin-ember-suave": "^1.0.0",
+    "eslint": "^6.6.0",
```

Comment:
Oh, I missed before that we have `eslint` as a direct dependency. How about we get rid of `ember-cli-eslint` then so we don't have two versions kicking around?

Reply:
I think the opposite it what we want, but I went ahead and remove eslint, so we can closer align to Ember projects.

Reply:
Oh, okay. Everything I've heard indicates that `ember-cli-eslint` is going to be removed from the default blueprint now that we have `lint:js` in the travis config. But it hasn't yet, so I'm fine doing it this way.